docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor.md

```diff
@@ -74,27 +74,25 @@ In this step, you use a Sumo-provided Azure Resource Manager (ARM) template to c
 
 Follow these steps to export metrics for a resource to Event Hub.
 
-1. From the left pane, select **ALL Services.**
-1. Search for and select "Monitor".
-1. In the **Monitor** pane, select **Diagnostic Settings** under **Settings**.
-1. Select the resource for which you want to export metrics. If **diagnostics** is not enabled, click **Turn on Diagnostics Settings.**
-1. Once diagnostics are enabled, click **Add a diagnostic setting**.
+1. In the Resource Group window, search for “SMNamespace” in “Resources” Tab. Click on `“SMNamespace<random-string>”`.
+1. From the left pane, select **Diagnostic Settings** under **Monitoring**.
```

Comment:
this is all incorrect you do not go to diagnostic settings of event hub namespace, you have to go to the diagnostic settings of the service which you want to monitor

Reply:
Updated